_posts/2016-09-12-schedule.md

```diff
@@ -0,0 +1,25 @@
+---
+layout: post
+title: "Schedule Released!"
+authors:
+ - badboy
+ - hoverbear
+---
+
+At long last, a schedule!
+
+We're proud to announce our lineup for RustFest 2016 in Berlin!
+
+[**Click here to view the schedule!**](/schedule)
+
+# Saturday
+
+Starting at 9:00 and going until 19:00, Saturday is full of talks at the Microsoft Atrium!
+
+** There will be dinner on Saturday!**
+
+# Sunday
+
+Starting at 10:00 and going until 17:30, Sunday features keynotes to start and end the day as well as workshops start!
+
+Workshops are located at Co.up and ThoughtWorks Werkstatt which are a short transit trip. You can check out the [transport](blog/Transportation-at-the-Conference) post to see how to get to them!
```

Comment:
should make that an absolute link, to be sure. Not sure this would work from inside `/blog/this-post`.